frontend/src/app/pages/settings-page/settings-page.component.html

```diff
@@ -1,72 +1,52 @@
+            Change your account data
+        </mat-card-subtitle>
         <mat-card-content>
-            <mat-accordion class="align-accordion-descriptions">
-                <mat-expansion-panel>
-                    <mat-expansion-panel-header>
-                        <mat-panel-title>
-                            User data
-                        </mat-panel-title>
-                        <mat-panel-description>
-                            Basic user information
-                        </mat-panel-description>
-                    </mat-expansion-panel-header>
-
-                    <mat-form-field>
-                        <input matInput placeholder="First name" type="text" [formControl]="userFirstName" required>
-                        <mat-error *ngIf="userFirstName.invalid">{{getUserFirstNameErrorMessage()}}</mat-error>
-                    </mat-form-field>
-                    <mat-form-field>
-                        <input matInput placeholder="Last name" type="text" [formControl]="userLastName" required>
-                        <mat-error *ngIf="userLastName.invalid">{{getUserLastNameErrorMessage()}}</mat-error>
-                    </mat-form-field>
-                    <mat-form-field>
-                        <input matInput placeholder="E-mail" type="text" [formControl]="userEmail" required readonly>
-                        <mat-error *ngIf="userEmail.invalid">{{getUserEmailErrorMessage()}}</mat-error>
-                    </mat-form-field>
-
-                    <button mat-button color="primary" (click)="updateUserDetails()" [disabled]="!validateUserInput()">
-                      Save
-                    </button>
-
-                </mat-expansion-panel>
-            </mat-accordion>
+            <mat-form-field>
+                <input matInput placeholder="First name" type="text" [formControl]="userFirstName" required>
+                <mat-error *ngIf="userFirstName.invalid">{{getUserFirstNameErrorMessage()}}</mat-error>
+            </mat-form-field>
+            <mat-form-field>
+                <input matInput placeholder="Last name" type="text" [formControl]="userLastName" required>
+                <mat-error *ngIf="userLastName.invalid">{{getUserLastNameErrorMessage()}}</mat-error>
+            </mat-form-field>
+            <mat-form-field>
+                <input matInput placeholder="E-mail" type="text" [formControl]="userEmail" required readonly>
+                <mat-error *ngIf="userEmail.invalid">{{getUserEmailErrorMessage()}}</mat-error>
+            </mat-form-field>
+            <button mat-button color="primary" (click)="updateUserDetails()" [disabled]="!validateUserInput()">
+                Save
+            </button>
         </mat-card-content>
     </mat-card>
-    <mat-card *ngIf="currentUser.role === 'admin'">
-        <mat-card-header>
-            <mat-card-title><h1>Administrator Panel</h1></mat-card-title>
-        </mat-card-header>
+    <mat-card class="tab-container__card">
```

Comment:
Please leave if-statement for admin role as it was done previously:
```suggestion
    <mat-card class="tab-container__card" *ngIf="currentUser.role === 'admin'">
```